run-and-report.py

```diff
@@ -91,7 +94,10 @@ def command_name(command_array):
     if options.ttl:
         riemann_event["ttl"] = int(options.ttl)
     riemann_event["host"]  = socket.gethostname()
-    riemann_event["metric"] = duration
+    if options.stdout:
```

Comment:
is the option still called `stdout` even though the flag is now `metric-from-stdout`?